buster/documents_manager/service.py

```diff
@@ -76,3 +80,19 @@ def delete_source(self, source: str) -> tuple[int, int]:
         self.index.delete(filter={"source": source}, namespace=self.namespace)
 
         return source_deleted, documents_deleted
+
+    def drop_db(self):
+        """Drop the currently accessible database.
+
+        For Pinecone, this means deleting everything in the namespace.
+        For Mongo DB, this means dropping the database. However this needs to be done manually through the GUI.
+        """
+        confirmation = input("Dropping the database is irreversible. Are you sure you want to proceed? (Y/n): ")
```

Comment:
Nitpick , but it should be N/y since No is by default

Reply:
done